src/snowflake/cli/plugins/containers/compute_pool/manager.py

```diff
@@ -1,19 +1,33 @@
+from typing import Optional
+
 from snowflake.cli.api.sql_execution import SqlExecutionMixin
+from snowflake.cli.plugins.containers.common import strip_empty_lines
 from snowflake.connector.cursor import SnowflakeCursor
 
 
 class ComputePoolManager(SqlExecutionMixin):
     def create(
-        self, pool_name: str, num_instances: int, instance_family: str
+        self,
+        pool_name: str,
+        num_instances: int,
+        instance_family: str,
+        auto_resume: bool,
+        initially_suspended: bool,
+        auto_suspend_secs: int,
+        comment: Optional[str],
     ) -> SnowflakeCursor:
-        return self._execute_query(
-            f"""\
+        query = f"""\
             CREATE COMPUTE POOL {pool_name}
             MIN_NODES = {num_instances}
```

Comment:
num_instances is incorrect. We should use two arguments max_nodes and min_nodes.